### PR TITLE
[SEM-001] - Typo Besley

### PR DESCRIPTION
**Pull request by JersonEM**
I have created a component for the use of the Besley typeface plus its respective user manual:
- styles\global.css
- README_Estilos.md

### CODE REVIEW

**Review by Ditmar — CHANGES_REQUESTED** [review]
Something is wrong with the rebase please check it

File: styles/global.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=Besley:wght@400;700&display=swap');
```
Comment (Ditmar) [inline styles/global.css:1]: this font should be in another file called fonts. please use embedded fonts approach

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments

File: styles/global.css (hunk 1 above)
Comment (JersonEM) [reply styles/global.css]: Good night engineer has already been modified to the requirements

**Review by Ditmar — APPROVED** [review]

**Pull request merged**